packages/react/src/UnderlineNav/UnderlineNav.figma.tsx

```diff
@@ -28,22 +28,26 @@ figma.connect(
     props: {
       label: figma.textContent('Label'),
       current: figma.enum('state', {
-        selected: '"page"',
```

Comment:
We need double quotes, otherwise we end up with no quotes in figma.

Reply:
I think we can ignore the error here, as the output is fine, right?

![CleanShot 2024-07-03 at 09 22 45@2x](https://github.com/user-attachments/assets/8822683e-c360-460f-b731-b87dca76e2d4)

Reply:
It seems this was fixed on figmas side